python/nano/src/bigdl/nano/quantization/neural_compressor/quantization.py

```diff
@@ -96,10 +96,9 @@ def __init__(self):
                         to the new local class to avoid that.
                         """
                         self.metric = metric
-
                 self.metric = Metric(
                     MyMetric,
-                    name="%s_%s" % (framework_metric.__name__, type(metric).__name__)
+                    name=f"nano_metric_{framework_metric.get_next_metric_id()}"
```

Comment:
give it a more meaningful name

Reply:
yeah, maybe a metric name and an unique id.

Reply:
OK, let me roll back the name.